diff -up lua-5.4.4/Makefile.5~ lua-5.4.4/Makefile
--- lua-5.4.4/Makefile.5~	2022-03-05 02:10:02.539675745 +0100
+++ lua-5.4.4/Makefile	2022-03-05 02:10:48.977321966 +0100
@@ -54,6 +54,9 @@ all:	$(PLAT)
 $(PLATS) help test clean:
 	@cd src && $(MAKE) $@
 
+test_pgo: dummy
+	src/lua scimark.lua
+
 install: dummy
 	cd src && $(MKDIR) $(INSTALL_BIN) $(INSTALL_INC) $(INSTALL_LIB) $(INSTALL_MAN) $(INSTALL_LMOD) $(INSTALL_CMOD)
 	cd src && $(INSTALL_EXEC) $(TO_BIN) $(INSTALL_BIN)
@@ -101,6 +104,6 @@ pc:
 	@echo "includedir=$(INSTALL_INC)"
 
 # Targets that do not create files (not all makes understand .PHONY).
-.PHONY: all $(PLATS) help test clean install uninstall local dummy echo pc
+.PHONY: all $(PLATS) help test test_pgo clean install uninstall local dummy echo pc
 
 # (end of Makefile)
