diff -up lua-5.4.6/src/Makefile.1~ lua-5.4.6/src/Makefile
--- lua-5.4.6/src/Makefile.1~	2023-02-03 11:43:14.000000000 +0100
+++ lua-5.4.6/src/Makefile	2023-06-06 19:50:04.047831467 +0200
@@ -32,6 +32,7 @@ CMCFLAGS=
 
 PLATS= guess aix bsd c89 freebsd generic ios linux linux-readline macosx mingw posix solaris
 
+LUA_SO= liblua.so.$(V)
 LUA_A=	liblua.a
 CORE_O=	lapi.o lcode.o lctype.o ldebug.o ldo.o ldump.o lfunc.o lgc.o llex.o lmem.o lobject.o lopcodes.o lparser.o lstate.o lstring.o ltable.o ltm.o lundump.o lvm.o lzio.o
 LIB_O=	lauxlib.o lbaselib.o lcorolib.o ldblib.o liolib.o lmathlib.o loadlib.o loslib.o lstrlib.o ltablib.o lutf8lib.o linit.o
@@ -44,7 +45,7 @@ LUAC_T=	luac
 LUAC_O=	luac.o
 
 ALL_O= $(BASE_O) $(LUA_O) $(LUAC_O)
-ALL_T= $(LUA_A) $(LUA_T) $(LUAC_T)
+ALL_T= $(LUA_A) $(LUA_T) $(LUAC_T) $(LUA_SO)
 ALL_A= $(LUA_A)
 
 # Targets start here.
@@ -60,6 +61,11 @@ $(LUA_A): $(BASE_O)
 	$(AR) $@ $(BASE_O)
 	$(RANLIB) $@
 
+$(LUA_SO): $(BASE_O)
+	gcc -Wl,-soname=liblua.so.$(V) -shared -o $@ $(MYLDFLAGS) $^ $(LIBS)
+	rm -f liblua.so
+	ln -s $@ liblua.so
+
 $(LUA_T): $(LUA_O) $(LUA_A)
 	$(CC) -o $@ $(LDFLAGS) $(LUA_O) $(LUA_A) $(LIBS)
 
