diff -up lua-5.4.4/src/luaconf.h.3~ lua-5.4.4/src/luaconf.h
--- lua-5.4.4/src/luaconf.h.3~	2022-03-05 02:09:05.632334350 +0100
+++ lua-5.4.4/src/luaconf.h	2022-03-05 02:09:47.782105908 +0100
@@ -327,6 +327,12 @@
 ** ===================================================================
 */
 
+#ifndef LUA_NO_COMPAT
+#define LUA_COMPAT_5_3 1
+#define LUA_COMPAT_5_2 1
+#define LUA_COMPAT_5_1 1
+#endif
+
 /*
 @@ LUA_COMPAT_5_3 controls other macros for compatibility with Lua 5.3.
 ** You can define it to get all options, or change specific options
